zuul.d/projects.yaml

```
@@ -5,6 +5,11 @@
     name: ^(github.com/|)ansible-collections/.*
     default-branch: main
 
+- project:
+    name: ^(github.com/|)ansible-collections/community\..*
+    templates:
+      - publish-to-galaxy
```

```suggestion
      - publish-to-galaxy-3pci
```

Let's use 3pci since that's that the projects had before, i.e:
https://github.com/ansible/project-config/blob/278bdd2e6863b53804995c5d129a02e47859ad36/zuul.d/projects.yaml#L240-L261

The 3pci variant doesn't add check/gate jobs.